docs/reference/announcements/870.md

```diff
@@ -0,0 +1,136 @@
+
+For SaaS customers, identity management in Camunda 8.7 remains consistent with Camunda 8.6, allowing the attachment of a single IdP per organization. However, cluster-level identity capabilities are provided for SaaS as well as Self-Managed. This means that user groups, roles and access permissions can now be managed at the cluster level, giving SaaS customers the same granular access control as in Self-Managed environments.
+
+### Decoupling from Keycloak <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
+
+The built-in Keycloak integration is removed for Self-Managed, allowing customers to use any compatible identity provider (IdP).
+
+- Keycloak remains fully supported as an external option.
+- OpenID Connect (OIDC) remains the standard for seamless integration with chosen IdPs.
+
+### Resource-based permissions
+
+Resource-level permissions are introduced for process definitions and web applications.
+
+- Admin users retain full access, but regular users must be granted specific permissions to perform actions/view resources.
+- For organizations that build custom front-ends and access Camunda via API, users with API permissions can still access process data through the V2 API.
+
+## Installation and deployment updates <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
+
+Camunda 8.7 introduces a streamlined architecture, consolidating core components like Zeebe, Operate, and Tasklist into a single deployable unit, simplifying deployment and management. Enhanced deployment options are also included, such as new Kubernetes Helm guides, deployment reference architectures, and improved support for professional developers with Camunda 8 Run.
+
+You can download the alpha release of the unified package as an executable Java application (Camunda Orchestration Core) or a Docker image, which is available in our GitHub Repository.
+
+### Helm charts
+
+If you are using the recommended Camunda 8 deployment option (Helm charts), the upgrade path from version 8.6 to 8.7 will be straightforward. Updated Helm charts will be provided to support the upgrade to the new streamlined architecture.
+
+New migration guides will also be provided to support you when migrating from a previous Camunda version.
+
+:::caution
+Additional upgrade considerations are necessary for deployments that use custom scripts, such as Docker containers, manual installations, or custom-developed Kubernetes deployments. For these deployments, customers can either continue to deploy with their original 8.6 topology and upgrade each component independently, or adopt our Helm Chart approach for the upgrade, which allows for unifying the deployment into a single JAR or container executable.
+:::
+
```

Comment:
Could we add another section:
### Camunda 8.7 manual installation
For organizations that do not use cloud-native platforms like Kubernetes or container services, we will publish a reference architecture that provides guidance on implementing Camunda production clusters on VM-based systems, using AWS EC2 as an example. The architecture will include details on optimal instance sizing, network configurations, and security best practices to ensure robust performance and reliability.

Reply:
Added 👍